### ISSUE TITLE
Customize Reported CarPlay Name

### ISSUE THREAD

**Issue by schaffman5** [superdavex/Sony-XAV-AX100#17]
I'm wondering if it's possible to customize the reported name of the head unit.  Reason for asking is I have one in my car and one in wife's, and both appear the same in the iPhone menu.  Would be cool to indicate the car where it's installed instead of just "XAV-AX100".

![carplay_iphone](https://user-images.githubusercontent.com/1555095/41814560-e1065778-771e-11e8-8ff4-71cecd9ba609.png)

BTW, thanks for sharing this code.  All works perfectly on both units.

**Comment by prgmaticoptimst** [comment]
I second this. Although really iOS should allow you change these names to whatever. I have 2 I’d these headunits as well and would be a nice feature.

**Comment by misterblack1** [comment]
Yes on Android you can just go into Settings -> Bluetooth, selected the
paired device and rename it to whatever you want. I do this for the same
reason. Two different cars with the same XAV-AX100 headunit.
